shared/checks/oval/audit_rules_privileged_commands.xml

```diff
@@ -31,22 +31,15 @@
   <!-- First define OVAL entities that can be reused across tests below -->
   <unix:file_object id="object_system_privileged_commands" comment="system files with setuid or setgid permission set" version="1">
     <unix:behaviors recurse="directories" recurse_direction="down" recurse_file_system="local" max_depth="-1" />
-    <unix:path operation="equals">/</unix:path>
-    <!-- [a-z]+ regex below is a workaround for OpenSCAP https://fedorahosted.org/openscap/ticket/457 bug -->
-    <unix:filename operation="pattern match">[a-z]+</unix:filename>
+    <unix:filepath operation="pattern match">^\/(?!proc|sbin|lib|lib64|dev|sys|run|var\/lib\/nfs|var\/run\/user\/1000\/gvfs).*\/[a-z]+</unix:filepath>
     <filter action="include">state_setuid_or_setgid_set</filter>
-    <filter action="exclude">state_dev_proc_sys_dirs</filter>
```

Comment:
This excludes filter doesn't appear to be working. Any excludes filter for that matter.

Reply:
This is very likely a bug in openscap, see https://www.redhat.com/archives/open-scap-list/2018-April/msg00001.html